RecoMuon/MuonIsolation/plugins/MuonPFIsolationWithConeVeto.cc

```diff
@@ -59,11 +59,11 @@ bool MuonPFIsolationWithConeVeto::isInIsolationCone(const reco::CandidatePtr& ph
           break;
         }
       }
-      result *= ( is_vertex_allowed );
+      result = is_vertex_allowed;
```

Comment:
I think ```result = is_vertex_allowed;``` and ```result = result && is_vertex_allowed;``` should be identical here as ```result``` is always true at this point. but login at line number 64 should be changed.